app/src/main/java/com/github/libretube/repo/LocalFeedRepository.kt

```diff
@@ -81,31 +81,32 @@ class LocalFeedRepository : FeedRepository {
         if (channelIds.isEmpty()) return
 
         val totalExtractionCount = AtomicInteger()
-        val chunkedExtractionCount = AtomicInteger()
+        val channelExtractionCount = AtomicInteger()
```

Comment:
We could possibly add some small comments about the different atomic integers and their difference here, but that's not that important